src/random.cr

```diff
@@ -105,13 +119,27 @@ module Random
     end
   end
 
+  # Returns a random `Float64` in the given *range*.
+  #
+  # ```
+  # Random.new.rand(6.2..21.768) # => 15.2989
+  # ```
+  def rand(range : Range(Float, Float)) : Float64
+    span = range.end - range.begin
+    if span > 0 || !range.excludes_end?
+      range.begin + (range.excludes_end? ? rand(span) : rand * span)
+    else
+      raise ArgumentError.new "incorrect rand value: #{range}"
+    end
+  end
+
   # see `#rand`
   def self.rand : Float64
     DEFAULT.rand
   end
 
   # see `#rand(x)`
-  def self.rand(x) : Int32
+  def self.rand(x)
```

Comment:
Got it:)
@AlexWayfer I have to appologize)
@asterite Are there any other concerns to get this PR merged?